README.md

```diff
@@ -8,25 +8,54 @@ This is the backend package for the Hourai Teahouse website.  Thus far it was wr
 
 ## Setup
 
-0. Install python 3.5.3 or newer.
-0. Optionally use virtualenv to set up a virtual enviroment for the project.
-0. Use ``pip install -r requirements.txt`` to install the relevant dependencies.
+To get started, you will need to install python 3.5 or newer.
+
+You can set up the backend in your working environment or create a virtual
+environment for the backend that is isolated from your current environment.
+If you do not wish to use a virtual environment, you can skip to the
+installation steps below.
+
+If you choose to use a virtual environment, you will have to install the
+virtualenv utility and use it to create a environment that uses Python 3.5.
+Then, activate the environment and continue with the installation steps below.
+
+Installation Steps:
+
+1. Install pip for python 3.
+2. Use `pip install -r requirements.txt` to install the relevant dependencies.
+3. Append the absolute path of src/houraiteahouse to the PYTHONPATH environment
+variable.
 
 ### Setting up the database
 
-To set up the database, follow standard SQLAlchemy setup procedures.  Use PYTHONPATH src/houraiteahouse:
-* Setup/configure mysql (or whatever you're using instead).  You can optionally use manage.py db create_db and other options to simplify this process.
-* Run "manage.py db init" to initialize the Flask migrations directory
-* Run "manage.py db migrate" to generate DB config from models
-* Run "manage.py db upgrade" to commit changes to the database
+* Setup/configure mysql (or whatever compatible DMBS you are using instead).
+* Create the file "secretkey" in /var/htwebsite. It simply needs to exist. 
+* Create the file "news" in /var/htwebsite. It simply needs to exist.
+* Create the file "mysqlcreds" in /var/htwebsite. In this file, on three
+separate lines, specify, in this order: DBMS username, DBMS password, DBMS
+database name.
+* You may be able to use `manage.py create_db` and other options to simplify this process.
```

Comment:
Probably best to move this before the other two lines, and provide an example command on how to run it.